18. Lấy số lượng Blog active của user có id là 1,2,4.md

```diff
@@ -0,0 +1,7 @@
+### 18. Lấy số lượng Blog active của user có id là 1,2,4
+```mysql
+select  user_id, count(*), is_active
+from blog
+group by user_id
+having  user_id in (1, 2, 4)  and is_active = 1;  
```

Comment:
tại sao anh lại dùng `having  user_id in (1, 2, 4) ` là sao ạ?

Reply:
vi nguoi ta tim user_id 1,2,4. thi co danh sach 3 thang user_id do